app/models/tag.rb

```diff
@@ -33,4 +36,10 @@ def self.parse(tag_string)
     # FIXME: Doesn't exist on topo - but blows up when there are nil values in the db.
     tag_params.transform_values { |e| e || "" }
   end
+
+  private
+
+  def update_portfolio_stats
+    portfolio&.update_statistics
```

Comment:
@gmcculloug I think this would have to run thru all portfolios. Since we have this
has_many :portfolios, :through => :portfolio_tags